happi/cli.py

```diff
@@ -154,11 +155,9 @@ def search_parser(
                 criteria = 'name'
                 value = user_arg
             if criteria in client_args:
-                logger.error(
-                    'Received duplicate search criteria %s=%r (was %r)',
-                    criteria, value, client_args[criteria]
+                raise click.ClickException(
+                     f"Received duplicate search criteria {criteria}={value!r} (was {client_args[criteria]!r})"
```

Comment:
Some of these lines are rather long.  I'm surprised they passed flake8, but I realize line lengths can be a bit flexible.